docs/cmdlet-example/README.md

```diff
@@ -0,0 +1,118 @@
+# Building a C# Cmdlet for PowerShell Core
```

Comment:
It's better to say "with Visual Studio 2017", to differentiate from #5117

Reply:
updated